rasa/core/events/__init__.py

```diff
@@ -180,19 +180,21 @@ class UserUttered(Event):
 
     def __init__(
         self,
-        text,
+        text: Optional[Text] = None,
```

Comment:
We handled the case `UserUttered` to `Dict`. How about the case `Dict` to `UserUttered`?

Reply:
Good! So will need to create a method similar to this, passing a dict as a parameter:
https://github.com/RasaHQ/rasa/blob/7c9fd15a6f2b3e393d8a9359eb3934195b64b8d9/rasa/core/events/__init__.py#L211
Or is there a better way to handle Dict=>UserUttered?

Reply:
The `Dict->UserUttered` thing currently handled in `_from_story_string`. So you just have to add your stuff there.